app_dart/lib/src/request_handlers/check_for_waiting_pull_requests.dart

```diff
@@ -110,6 +111,41 @@ class CheckForWaitingPullRequests extends ApiRequestHandler<Body> {
     return result.data;
   }
 
+  Future<List<dynamic>> _queryCirrusGraphQL(
```

Comment:
This seems like it has potential to live in a shared file

Reply:
Kept that part in refresh_cirrus_status.dart only, and referenced it from check_for_waiting_pull_requests.dart.